src/sentry/lang/native/symbolicator.py

```diff
@@ -62,6 +62,9 @@
         "orgId": {"type": "integer"},
         "orgName": {"type": "string", "minLength": 1, "maxLength": 512},
         "encrypted": {"type": "string"},
+        "itunesCreated": {"type": "string"},
+        "itunesPassword": {"type": "string"},
+        "appconnectPrivateKey": {"type": "string"},
```

Comment:
@flub will remove the `encrypted` prop this week

Reply:
Yeah, they already exist here now (https://github.com/getsentry/sentry/blob/0ed459cc4cca0504f9de2cf80bbf8a3d7fb9fd21/src/sentry/api/endpoints/project_app_store_connect_credentials.py#L256-L257) and encrypted will be removed entirely because it serves no purpose anymore